### ISSUE TITLE
role changes default cron log path

### ISSUE THREAD

**Issue by jamescassell** [MindPointGroup/RHEL7-STIG#121]
```
TASK [R7S : MEDIUM | RHEL-07-021100 | PATCH | Cron logging must be implemented.] **
--- before: /etc/rsyslog.conf (content)
+++ after: /etc/rsyslog.conf (content)
@@ -61,7 +61,7 @@


 # Log cron stuff
-cron.*                                                  /var/log/cron
+cron.* /var/log/cron.log

 # Everybody gets emergency messages
 *.emerg                                                 :omusrmsg:*
```

The system as-shipped satisfies the STIG requirement of logging cron.  It is not required to have it in "cron.log" as the file name.

**Comment by mikerenfro** [comment]
PR #129 should resolve this.

**Comment by shepdelacreme** [comment]
Fixed in #129
